thunder/benchmarks/targets.py

```diff
@@ -937,3 +938,42 @@ def test_interpreter_nanogpt_gpt2_fwd(benchmark, executor: Callable):
     fn = wrap_for_benchmark(fn)
 
     benchmark.pedantic(fn, setup=setup, rounds=5, warmup_rounds=1)
+
+
+#
+# vision benchmarks
+#
+
+
+@pytest.mark.parametrize(
+    "executor,",
+    fwd_executors,
+    ids=fwd_executor_ids,
+)
+def test_resnet50_fwd(benchmark, executor: Callable):
+    b = ResNet50Benchmark((64, 3, 224, 224), device="cuda:0", dtype=torch.bfloat16, requires_grad=False)
```

Comment:
A parametrization of the batch size would be useful here. As I understand the number of channels and width and height is usually, at least in MLPerf benchmarks `3, 224, 224`, right?

Reply:
Yes, 224x224 is "the" imagenet input shape, and many models make the assumption that this is the size / minimal size. People have been using larger images, in particular for detection models, but for classification this seems to still be the benchmark.